### PR TITLE
github: also push to ghcr.io

### PR DESCRIPTION
**Pull request by budimanjojo**
Fixes: https://github.com/lldap/lldap/issues/527

The requested `ghcr.io/nitnelave/lldap` container image is commented out and can be uncommented if needed because it needs to have the repo owner to create a secret named `NITNELAVE_GITHUB_TOKEN`.

### CODE REVIEW

File: .github/workflows/docker-build-static.yml [hunk 1]
```diff
@@ -473,6 +473,20 @@ jobs:
...
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to GitHub Container Registry
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # - name: Login to nitnelave GitHub Container Registry
```
Comment (nitnelave) [inline .github/workflows/docker-build-static.yml:483]: That one should not be needed, the login above should be enough. Maybe it only works with nitnelave though?

File: .github/workflows/docker-build-static.yml [hunk 2]
```diff
@@ -484,7 +498,10 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           platforms: linux/amd64,linux/arm64
           file: ./.github/workflows/Dockerfile.ci.alpine
-          tags: nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
+          tags: |
+            nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
```
Comment (nitnelave) [inline .github/workflows/docker-build-static.yml:502]: I configured an "lldap" organization on dockerhub as well, can we add lldap/lldap here and below?

**Review by nitnelave — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-build-static.yml [hunk 3]
```diff
@@ -512,7 +532,10 @@ jobs:
...
+          tags: |
+            nitnelave/lldap:stable, nitnelave/lldap:stable-alpine, nitnelave/lldap:v${{ steps.slug.outputs.version-semantic }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}.${{ steps.slug.outputs.version-minor }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}.${{ steps.slug.outputs.version-minor }}.${{ steps.slug.outputs.version-patch }}, nitnelave/lldap:v${{ steps.slug.outputs.version-semantic }}-alpine, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}-alpine, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}-alpine.${{ steps.slug.outputs.version-minor }}-alpine, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}.${{ steps.slug.outputs.version-minor }}.${{ steps.slug.outputs.version-patch }}-alpine
```
[the remark this answers is not in the document]
Comment (budimanjojo) [reply .github/workflows/docker-build-static.yml:532]: yeah it is :D

File: .github/workflows/docker-build-static.yml (hunk 1 above)
Comment (budimanjojo) [reply .github/workflows/docker-build-static.yml]: okay I updated to only login using nitnelave, not sure if it'll work but you can try it out

File: .github/workflows/docker-build-static.yml (hunk 2 above)
Comment (budimanjojo) [reply .github/workflows/docker-build-static.yml:496]: done!

**Comment by budimanjojo** [comment]
@nitnelave I've updated the commit, please have another look :D

File: .github/workflows/docker-build-static.yml [hunk 4]
```diff
@@ -484,7 +492,11 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           platforms: linux/amd64,linux/arm64
           file: ./.github/workflows/Dockerfile.ci.alpine
-          tags: nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
+          tags: |
+            nitnelave/lldap:latest, nitnelave/lldap:latest-alpine
+            lldap/lldap:latest, lldap/lldap:latest-alpine
+            ghcr.io/${{ github.repository_owner }}/lldap:latest, ghcr.io/${{ github.repository_owner }}/lldap:latest-alpine
```
[the remark this answers is not in the document]
Comment (budimanjojo) [reply .github/workflows/docker-build-static.yml:498]: the doc is not mentioning anything about organization repo, maybe I should use `github.repository` instead, it's also will make it shorter

**Comment by budimanjojo** [comment]
Changed `${{ github.repository_owner }}/lldap` to `${{ github.repository }}` instead

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Sadly, the only thing we can do to try this out is to merge it :)
It looks good, I'll merge it when I can. It'll only have `latest` until the next stable release, though.

**Comment by budimanjojo** [comment]
no worries man :D

**Comment by budimanjojo** [comment]
@nitnelave I see that the test failed with `403 Forbidden`. Maybe you really need to have the `docker login` step for your `lldap` account?

**Comment by nitnelave** [comment]
Yeah, I saw that. I think the issue is with ghcr.io/nitnelave/lldap, which we don't actually need (for ghcr.io, we can transition everyone directly to lldap/lldap)

We're trying to fix this (and reduce the boilerplate) in https://github.com/lldap/lldap/pull/532

**Pull request merged**